### DISCUSSION TITLE
Translation of the Frontend

### DISCUSSION THREAD

**Discussion by msmedien** [photobooth-app/photobooth-app#209] [Q&A]
Hi,

where can i edit the text for the frontend? I would like to translate it to german, until a "real" translation-feature is available in the admin-panel.

Thanks for your help! :-)
Marcus

**Comment by mgineer85** [comment]
I don't think that it is really feasible to translate as is.
It is hardcoded right now in the photobooth-frontend. so you would need to edit that project and build it. The built files you add to the webserver then.
You could also try to upgrade to the latest beta if you want to. The video feature does not yet work 100% but you can disable for now and have the updated frontend including translation.

**Comment by msmedien** [comment]
Thanks for yor reply and help! :)

How can i install the latest beta? I used the pipx method. Do i manage it with this command?

`pip upgrade git+https://github.com/photobooth-app/photobooth-app.git@main`

Sorry for this question, but i'm not this good in Linux-Systems... :-/

**Comment by mgineer85** [comment]
You can check that page: https://photobooth-app.org/setup/update/#method-a-update-with-pipx   and the latest section. I did not upgrade with pipx to github repo myself, but this should help you to find the direction.

**Comment by msmedien** [comment]
Thanks, but i doesn't work... :-/

If i use the commands `pip upgrade git+https://github.com/photobooth-app/photobooth-app.git@main` or the other option to dev-branch `pip install --upgrade --force-reinstall --no-deps git+https://github.com/photobooth-app/photobooth-app.git@dev` i'm getting just error messages... "externally managed envoirment" Python is not installed system-wide.

I think i need to wait for the regular update... Or do you know other solutions?

**Reply by mgineer85** [reply]
of course you have to stick to your installation methods program, that is pipx for you. 
check this out or google for pipx upgrade GitHub https://github.com/pypa/pipx/issues/892
